src/libs/testCrash/index.ts

```diff
@@ -0,0 +1,2 @@
+const testCrash = () => {}
```

Comment:
seems like you could easily add a test crash on web too, it would just be a JS-level crash instead of a native-level crash. JS crashes still go to Crashlytics